server/terraform/modules/myhealth/main.tf

```diff
@@ -10,6 +10,19 @@ terraform {
   required_version = ">= 0.12"
 }
 
+locals {
+  # We want the storage bucket to be multi-region. Lookup the location for that
+  # using var.region prefix; based on current list at:
+  # https://cloud.google.com/storage/docs/locations
+  storage_location_prefixes = {
+    "EUROPE" = "EU",
+    "US"     = "US",
+    "ASIA"   = "ASIA"
+  }
+  storage_bucket_location = lookup(local.storage_location_prefixes,
+  upper(split("-", var.region)[0]), var.region)
```

Comment:
Nice way to do this. We'll need to fix it up if we deployed in a region where there's no multi-region support, e.g. Australia... but mostly this will do the right thing.

Reply:
Exactly, if region is set to AUSTRALIA-SOUTHEAST1 then bucket location will also be AUSTRALIA-SOUTHEAST1.

Reply:
So it defaults to a single region if multi-region doesn’t exist? Document either way on that behaviour.